src/Jackett.Common/Indexers/ShizaProject.cs

```diff
@@ -45,170 +48,141 @@ internal class ShizaProject : BaseWebIndexer
-            var releases = new List<ReleaseInfo>();
-
-            try
-            {
-                var parser = new HtmlParser();
-                var document = await parser.ParseDocumentAsync(result.ContentString);
-
-                foreach (var linkNode in document.QuerySelectorAll(ReleaseLinksSelector))
-                {
-                    var url = linkNode.GetAttribute("href");
-                    releases.AddRange(await FetchShowReleases(url));
-                }
-            }
-            catch (Exception ex)
-            {
-                OnParseError(result.ContentString, ex);
-            }
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+            => query.IsTest || string.IsNullOrWhiteSpace(query.SearchTerm)
+            ? await FetchNewReleases()
+            : await PerformSearch(query);
 
-            return releases;
+        private  Task<IEnumerable<ReleaseInfo>> PerformSearch(TorznabQuery query)
+        {
+            // var title = _EpisodeRegex.Replace(query.SearchTerm, string.Empty).TrimEnd();
+            // var queryParameters = new NameValueCollection
+            // {
+            //     { "filter", "names,poster.url,code,torrents.list,season.year" },
+            //     { "limit", "100" }
+            // };
+            // var response = await RequestWithCookiesAndRetryAsync(Configuration.ApiLink.Value + "/searchTitles?" + queryParameters.GetQueryString() + "&search=" + Uri.EscapeDataString(title));
+            // if (response.Status != HttpStatusCode.OK)
+            //     throw new WebException($"AniLibria search returned unexpected result. Expected 200 OK but got {response.Status}.", WebExceptionStatus.ProtocolError);
+
+            // var results = ParseApiResults(response.ContentString);
+            // return results.Where(release => query.MatchQueryStringAND(release.Title, null, title));;
+
+            logger.Info("Search");
+            return null;
         }
 
-        private async Task EnsureAuthorized()
+        private class ReleasesResponse
         {
-            var result = await RequestWithCookiesAndRetryAsync(SiteLink);
+            public ReleaseContent Release { get; set; }
 
-            if (!IsAuthorized(result))
+            public class ReleaseContent
             {
-                await ApplyConfiguration(null);
+                public string Name { get; set; }
             }
         }
-
-        private async Task<List<ReleaseInfo>> FetchShowReleases(string url)
+        
+        private async Task<IEnumerable<ReleaseInfo>> FetchNewReleases()
         {
-            var releases = new List<ReleaseInfo>();
-            var uri = new Uri(url);
-            var result = await RequestWithCookiesAndRetryAsync(url);
-
-            try
+            logger.Info(GraphqlEndpointUrl);
+            using var graphQLClient = new GraphQLHttpClient(GraphqlEndpointUrl, new NewtonsoftJsonSerializer());
```

Comment:
To continue development I have to make GraphQL client works with proxy.

Constructor of GraphQLHttpClient could use HttpClient option to use configured HttpClient (https://github.com/graphql-dotnet/graphql-client/blob/master/src/GraphQL.Client/GraphQLHttpClient.cs#L66).
As far as I understands in indexer we should already have configured httpClient, so my question is how to get it to initialize GraphQLHttpClient?

@ngosang Maybe you can consult me as one of authors of https://github.com/Jackett/Jackett/blob/master/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs